README.md

```diff
@@ -1,68 +1,116 @@
-
-The build is minified and the filenames include the hashes.<br>
-Your app is ready to be deployed!
-
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
-
-### `npm run eject`
-
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
-
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
-
-Instead, it will copy all the configuration files and the transitive dependencies (Webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
-
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
-
+# Directory Front End
+
+This project is the front-end application for the [GCTools Directory](https://profile.gccollab.ca)
+
+## Motivation
+
+The front end app is a stand alone app and requires our [account](https://github.com/gctools-outilsgc/concierge) and [profile as a service](https://github.com/gctools-outilsgc/profile_service) apps in order to fully work. This is intended to work in our micro service architecture as shown here:
+
+![Image of OADW Architecture Concept](https://documentation.beta.gccollab.ca/static/OADW_Architecture-Walkthrough-a6bf66bae45a89a68cadee688d31d43c-71e8e.png)
+
+## Installation
+
+You need [nodejs](https://nodejs.org/en/), [yarn](https://yarnpkg.com/) and ideally [docker-compose](https://docs.docker.com/compose/install/)
+
+### Install for Dev
+If you have an account and profile service spun up you can make the connections in `scripts/start.js`
+
+```
+# scripts/start.js
+
+# your dev url
+const BASE_URL = "http://localhost:3000";
+# profile service endpoint
+if (!process.env.REACT_APP_GQL_ENDPOINT)
+  process.env.REACT_APP_GQL_ENDPOINT = 'https://paas.beta.gccollab.ca/';
+# account service endpoint
+if (!process.env.REACT_APP_OIDC_AUTHORITY)
+  process.env.REACT_APP_OIDC_AUTHORITY = 'https://dev.account.gccollab.ca/openid';
+# account client ID
+if (!process.env.REACT_APP_OIDC_CLIENT_ID)
+  process.env.REACT_APP_OIDC_CLIENT_ID = 123;
+```
+After you config the start script with your end points:
+```
+yarn install
+yarn start
+```
+
+### Developing with Docker
+
+If you do not have these services spun you can create a full end to end environment through 
+
+`docker-compose` up will deploy and configure to work together the following services:
+- Directory-fe on port 8008
+- Profile service
+- Notification service
+- Search
+- RabbitMQ
+- Account on port 8080
+
```

Comment:
the rest that I wanted to add after testing out a few scenarios:

You can hook up your `yarn start` dev instance to these local services by copying the environment variables set for the directory-fe container in docker-e2e/docker-compose.yaml into the respective variable in start.js as mentioned above.